Utilities/js.cpp

```diff
@@ -105,6 +105,21 @@ static ErrorOr<void> print(JS::Value value, PrintTarget target = PrintTarget::St
     return print(value, *stream);
 }
 
+static ErrorOr<void> print_all_arguments(JS::VM const& vm, PrintTarget target = PrintTarget::StandardOutput)
+{
+    auto stream = TRY(target == PrintTarget::StandardError ? Core::File::standard_error() : Core::File::standard_output());
+
+    for (size_t i = 0; i < vm.argument_count(); i++) {
+        TRY(print(vm.argument(i)));
+
+        if (i < vm.argument_count() - 1) {
+            TRY(stream->write_formatted(" "));
```

Comment:
`write_formatted` when we aren't formatting anything is a bit unnecessary. We can do this instead:
```c++
TRY(stream->write_until_depleted(" "sv));
```